testing/kokoro/continuous.cfg

```diff
@@ -13,4 +13,4 @@
 # limitations under the License.
 
 # Location of the build script in this repository.
-build_file: "cloud-profiler-nodejs/testing/integration_test.sh"
+build_file: "cloud-profiler-nodejs/prebuild_binaries/linux_build_all.sh"
```

Comment:
The name is misleading, it's not just build.

Reply:
Switched to "linux_build_and_test"